scipy/signal/tests/test_filter_design.py

```diff
@@ -1176,34 +1198,34 @@ def test_fs_param(self):
         # N = None, whole=False
         w1, h1 = freqz_zpk(z, p, k, whole=False, fs=fs)
         w2, h2 = freqz_zpk(z, p, k, whole=False)
-        assert_allclose(h1, h2)
-        assert_allclose(w1, np.linspace(0, fs/2, 512, endpoint=False))
+        xp_assert_close(h1, h2)
+        xp_assert_close(w1, np.linspace(0, fs/2, 512, endpoint=False))
 
         # N = None, whole=True
         w1, h1 = freqz_zpk(z, p, k, whole=True, fs=fs)
         w2, h2 = freqz_zpk(z, p, k, whole=True)
-        assert_allclose(h1, h2)
-        assert_allclose(w1, np.linspace(0, fs, 512, endpoint=False))
+        xp_assert_close(h1, h2)
+        xp_assert_close(w1, np.linspace(0, fs, 512, endpoint=False))
 
         # N = 5, whole=False
         w1, h1 = freqz_zpk(z, p, k, 5, fs=fs)
         w2, h2 = freqz_zpk(z, p, k, 5)
-        assert_allclose(h1, h2)
-        assert_allclose(w1, np.linspace(0, fs/2, 5, endpoint=False))
+        xp_assert_close(h1, h2)
+        xp_assert_close(w1, np.linspace(0, fs/2, 5, endpoint=False))
 
         # N = 5, whole=True
         w1, h1 = freqz_zpk(z, p, k, 5, whole=True, fs=fs)
         w2, h2 = freqz_zpk(z, p, k, 5, whole=True)
-        assert_allclose(h1, h2)
-        assert_allclose(w1, np.linspace(0, fs, 5, endpoint=False))
+        xp_assert_close(h1, h2)
+        xp_assert_close(w1, np.linspace(0, fs, 5, endpoint=False))
 
         # w is an array_like
         for w in ([123], (123,), np.array([123]), (50, 123, 230),
                   np.array([50, 123, 230])):
             w1, h1 = freqz_zpk(z, p, k, w, fs=fs)
             w2, h2 = freqz_zpk(z, p, k, 2*pi*np.array(w)/fs)
-            assert_allclose(h1, h2)
-            assert_allclose(w, w1)
+            xp_assert_close(h1, h2)
+            xp_assert_close(np.asarray(w), w1, check_dtype=False)
```

Comment:
this took me a second to process because `actual` and `desired` are the wrong way around - could you swap `np.asarray(w)` and `w1` around?

Reply:
Apparently not needed, so gone in the next commit. FWIW, for me `assert_close` should be symmetric (modulo a known thing about rtol, which only matters in rare edge cases).